suite-common/message-system/config/config.v1.json

```diff
@@ -1,8 +1,60 @@
 {
     "version": 1,
-    "timestamp": "2024-01-03T00:00:00+00:00",
+    "timestamp": "2025-01-06T00:00:00+00:00",
     "sequence": 74,
     "actions": [
+        {
+            "conditions": [
+                {
+                    "environment": {
+                        "desktop": ">=25.1.2",
+                        "mobile": ">=25.1.2",
+                        "web": ">=25.1.2"
+                    }
+                }
+            ],
+            "message": {
+                "id": "2855131c-c871-4fe3-a758-ab7bd92b9934",
+                "priority": 90,
+                "dismissible": false,
+                "variant": "info",
+                "category": ["banner"],
+                "content": {
+                    "en-GB": "Testnet account has been updated to new Testnet4.",
```

Comment:
Maybe more information is needed or the link to the article would be enough ?

Reply:
I wonder whether it is important to handle testnet update this much gracefully. Isn't testnet something that is unstable by default?

Reply:
Some people use Testnet for learning about self-custody Bitcoin. So they get confident before they buy some mainnet BTC. 
After this change they are going to see empty balance so some message is kind of nice to let them know that it is all right. Anyway the message will appear only to users with Testnet enabled.